src/program/mod.rs

```diff
@@ -470,12 +492,132 @@ impl<'a> Program<'a> {
         Ok(())
     }
 
+    /// Mode to input how many times to repeat a normal mode action
+    /// Previous input from normal mode is in `self.ui_state.current_input`
+    fn run_multi_normal_mode(&mut self) -> Result<(), String> {
+        use ui::MultiNormalAction;
+
+        let mut complete_action = false;
+        while !complete_action {
+            for event in self.screen.sdl_context.event_pump()?.poll_iter() {
+                let multi_action = ui::process_multi_normal_mode(&mut self.ui_state, event);
+
+                // Assume input is finished unless set by other actions
+                complete_action = true;
+
+                match multi_action {
+                    MultiNormalAction::MoreInput => {
+                        complete_action = false;
+                    }
+                    MultiNormalAction::Cancel => {
+                        self.ui_state.mode = Mode::Normal;
+                    }
+                    MultiNormalAction::Repeat(process) => {
+                        self.ui_state.process_action(process.action.clone());
+                        match process {
+                            ProcessAction {
+                                action: a,
+                                times: n,
+                            } => match (a, n) {
+                                (Action::Backspace, _) => {
+                                    // Divide repeat by int 10 to remove last digit
+                                    self.ui_state.repeat /= 10;
```

Comment:
@Davejkane I'm not sure on how you want this to perform, but in Vim if you backspace at any point it just clears the entire buffer rather than removing one of the numbers.